src/constants/chat.ts

```diff
@@ -45,6 +46,7 @@ export const modelMaxToken = {
   'gpt-4-32k-0314': 32768,
   'gpt-4-32k-0613': 32768,
   'gpt-4-1106-preview': 128000,
+  'gpt-4-0125-preview': 4096,
```

Comment:
@DigitallyTailored I think you have `modelMaxToken` incorrect here. Pretty sure it should be `128000`.

See also: https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo (`gpt-4-0125-preview` --> "Context Window").

Reply:
Oh nice - thanks! In the playground mine maxed at 4096 but I guess OpenAI hadn't updated that yet. Will patch this.

Reply:
@DigitallyTailored, there appears to have been a mix-up regarding the modelMaxToken setting for gpt-4-0125-preview. Your initial configuration of 4096 was indeed correct, reflecting the maximum output tokens for the GPT-4 Turbo series. The 128000 figure, as pointed out by @eshack94, pertains to the maximum input token length or the 'context window', rather than the output.

To ensure compatibility with the OpenAI API, it's essential that @ztjhz reverts the modelMaxToken value back to 4096. Any setting above this threshold will definitively cause an error.

Reply:
Thank you @NN1985  - I've reverted this now